ToolBox/AlgoSeekOptionsConverter/AlgoSeekOptionsConverter.cs

```diff
@@ -0,0 +1,164 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.IO;
+using QuantConnect.Logging;
+using QuantConnect.Data.Market;
+using System.Linq;
+using System.Diagnostics;
+using System.Threading.Tasks;
+
+namespace QuantConnect.ToolBox.AlgoSeekOptionsConverter
+{
+	public class AlgoSeekOptionsConverter
+	{
+		public static void ConvertToFineResolution(string sourceDirectory, string destinationDirectory, long flushInterval)
+		{
+			var quoteProcessor = new InMemoryDataProcessor(TickType.Quote, x => ((Tick) x).TickType == TickType.Quote);
+			var tradeProcessor = new InMemoryDataProcessor(TickType.Trade, x => ((Tick) x).TickType == TickType.Trade);
+
+			var files = Directory.EnumerateFiles(sourceDirectory).OrderByDescending(x => new FileInfo(x).Length);
+			var optionsReaders = files.Select(file => new AlgoSeekOptionsReader(file)).ToList();
+
+			var parsingTime = new Stopwatch();
+			var processingTime = new Stopwatch();
+			var diskTime = new Stopwatch();
```

Comment:
Would you still like each file to record it's progress, or just overall progress? I wasn't sure what we'd want here.